src/System.Runtime.CompilerServices.Unsafe/Directory.Build.props

```diff
@@ -5,6 +5,5 @@
     <AssemblyVersion>4.0.5.0</AssemblyVersion>
     <AssemblyKey>MSFT</AssemblyKey>
     <IsUAP>true</IsUAP>
-    <LanguageTargets Condition="'$(MSBuildProjectExtension)' == '.ilproj'">$(ToolsDir)\IL.targets</LanguageTargets>
```

Comment:
Are we going to delete the `IL.targets` from buildtools? It would be good to rip it out so we know that it is no longer being used.

Reply:
First, CoreCLR needs to move all of its ilproj to the SDK as well.

Reply:
CC. @AaronRobinsonMSFT